5.process_CP_features/scripts/0.merge_sc.py

```diff
@@ -29,11 +37,30 @@
 dest_datatype = "parquet"
 
 # s1lite directory
-source_dir = pathlib.Path("../../4.cellprofiler_analysis/analysis_output")
+source_dir = pathlib.Path("../../4.cellprofiler_analysis/analysis_output/").resolve(
+    strict=True
+)
 # directory where parquet files are saved to
-output_dir = pathlib.Path("../data/converted_data")
+output_dir = pathlib.Path("../data/0.converted_data")
 output_dir.mkdir(exist_ok=True, parents=True)
 
+if not in_notebook:
+    print("Running as script")
+    # set up arg parser
+    parser = argparse.ArgumentParser(description="Single cell extraction")
+
+    parser.add_argument(
+        "--well_fov",
+        type=str,
+        help="Path to the input directory containing the tiff images",
+    )
+
+    args = parser.parse_args()
+    well_fov = args.well_fov
+else:
+    print("Running in a notebook")
+    well_fov = "E-05_F0003"
```

Comment:
Just curious, as I might have missed it, is well_FOV specified here because you are running something like papermill or why have one FOV selected to run this process?